pkg/resourceutil/sort.go

```diff
@@ -99,3 +111,30 @@ func convertOrderingInMap(ordering resourceOrder) map[string]int {
 
 	return orderingMap
 }
+
+// returns the lowest order of the kinds specified in
+// mia-platform.eu/apply-before-kinds annotation or, if the annotation is not
+// present, returns the order specified in orderingMap for the resource's kind
+// P.S. we use decimals for overridden orders to avoid conflicts with defaults.
+func getOrderFromAnnotationOrKind(orderingMap map[string]int, resource Resource) (float32, bool) {
+	annotations := resource.Object.GetAnnotations()
+
+	if applyBeforeValue, applyBeforeFound := annotations[applyBeforeAnnotation]; applyBeforeFound {
+		order := float32(len(orderingMap))
+
+		for _, kind := range strings.Split(applyBeforeValue, ",") {
+			trimmedKind := strings.Trim(kind, " ")
```

Comment:
```suggestion
			trimmedKind := strings.TrimSpace(kind)
```
In this way, it remove all the trailing spaces removed (\n, \s, \t, ...)